hack/installers/checksums/add-helm-checksums.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+# Usage: ./add-helm-checksums.sh 3.9.4  # use the desired version
+
+set -e
+
+wget "https://get.helm.sh/helm-v$1-linux-amd64.tar.gz.sha256sum"   -O "helm-v$1-linux-amd64.tar.gz.sha256"
```

Comment:
Maybe a `for` loop on archs is suitable here ?

Reply:
done!